client/src/components/App.jsx

```diff
@@ -38,7 +38,8 @@ class App extends React.Component {
         <QuestionList
           data={this.state.questionList.QuestionListdata.QuestionList}
           currentProductID={this.state.currentProductID}
-          answerParser={this.state.questionList.QuestionListdata.answerParser}/>
+          answerParser={this.state.questionList.QuestionListdata.answerParser}
```

Comment:
In a future PR, maybe you can move this fn to the utilityFns. That way we can delete the sample data when we're ready for a production build

Reply:
Good call.